code/modules/research/nodes/biotech.dm

```diff
@@ -166,7 +166,7 @@
 
 /datum/technology/portable_biotech
 	name = "Portable Biotech"
-	desc = "Portable injection and scan method, capitalists' and not capitalists portable sleeper. Integrated medical sensors hud with hardsuits' systems."
+	desc = "Adds new configurations to portable sleeper. Integrated medical sensors hud with hardsuits systems."
```

Comment:
We haven't old configuration and portable sleeper, and we need to describe technologies, not designs